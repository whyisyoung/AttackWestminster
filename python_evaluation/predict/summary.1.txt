On Mar. 22, 2017 a terrorist attack occurred in London near Westminster Bridge. The police investigated that the attacker was Khalid Masood. During the terrorist attack the attacker killed 4 people and injured 50.
