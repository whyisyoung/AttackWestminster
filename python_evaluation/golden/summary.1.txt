On Wednesday March 22nd 2017, an attack took place at the Houses of Parliament in London.
A car was driven at high speed across Westminster Bridge - hitting victims as it went. The horror
began at the south London end of Westminster bridge when a driver in a Hyundai Tucson
jumped onto the pavement at around 2.30pm, mowing down pedestrians. He continued along
the pavement, plowing through at least a dozen victims in total, before coming to a stop at the
Houses of Parliament where he crashed his vehicle into railings. The driver then, armed with a
knife, tried to make his way into the Parliamentary estate. The assailant killed an unarmed
police officer before being shot dead by firearms officers just inside the gates. The attacker was
later identified as 52 year old Briton Khalid Masood. It is strongly believed that Masood was
inspired by Islamic extremism.

Five died in the rampage, including a police officer and more than 50 people were injured. The
five victims who lost their lives were PC Palmer, mother-of-two Aysha Frade, US tourist Kurt
Cochran, Romanian visitor Andreea Cristea, and pensioner Leslie Rhodes from south London.
Aysha Frade, who worked as a Spanish teacher, was making her way across Westminster
Bridge en route to collect her children from school when she was mowed down by the terrorist.
Frade had only just left her place of work at the nearby DLD College London and was on her
way to pick up her two children when fate intervened. Kurt Cochran was an American on
European tour with his wife. The man, in his 50s, is believed to have been killed while walking
on Westminster Bridge. Cochran was visiting London from Utah to celebrate his 25th
anniversary with his wife Melissa, who was among the injured. Romanian Andreea Cristea, 31,
who was visiting London with her boyfriend Andrei Burnaz, was knocked into the river Thames
after being struck by the car driven by Khalid Masood as she strolled along Westminster Bridge
with her boyfriend. She later died from her injuries. Leslie Rhodes, 75, a retired window cleaner,
was on the way to a hospital appointment at the time. The police officer killed was PC Keith
Palmer, 48, an unarmed police officer who was on duty with the Parliamentary and Diplomatic
command. The 52-year-old attacker was shot by armed police and was the first casualty to
arrive at St Mary's Hospital in Paddington, where he was pronounced dead. Among those
injured were people from Ireland, America, South Korea, Romania, France, Poland and
Germany. They were taken to St. Thomas' Hospital, which faces the Palace of Westminster
across the Thames and also to King's College Hospital and the Royal London Hospital.

Born in Kent, Masood was brought up in Rye, East Sussex and later attended secondary school
in Tunbridge Wells in Kent. Most recently he was living in the West Midlands. When he was 16,
he dropped out of school and by 18 he was described as a heavy drug user. In 2000, he was
sentenced to two years in prison for stabbing and slashing the face of a cafe owner at Northiam
in Sussex. In 2003, he was sentenced to a further six months in prison for possession of an
offensive weapon following another knife attack in Eastborne in Sussex. As well as these two
prison terms, Masood had convictions for public order offenses going back to 1983. Masood
was not known to be a religious person and frequently went to pubs.

Masood reportedly converted to Islam in 2005. He was described by police as a criminal with a
twenty year record of offending. He was born Adrian Russell Elms, but later changed the name
to Adrian Russell Ajao when he took the name of his stepfather. He changed his name to Khalid
Masood after he converted to Islam. The 52-year-old had recently split from his partner, Jane
Harvey, following bitter rows after she refused to allow her daughter to move from Kent. The
couple met at a Tunbridge Wells pub in 1991 before separating nine years later when he was
jailed. The couple had two daughters - now aged 19 and 24. The younger of the two still lives
with her mother, who is a director of a chemical company in Kent. The friend said the
tumultuous relationship was punctuated by Masood's violence towards others, who was then
known by his birth name. Masood masked his life of crime with a CV which claimed he was an
experienced English teacher who had worked across the world. The document lists Masood as
a university-educated English teacher with experience working in places such as Saudi Arabia
and Luton.

The Metropolitan Police initially believed the attack was inspired by international terrorism. On
March 23rd, The Amaq News Agency, affiliated with ISIS, took credit for the killings and
announced that the attacker was "a soldier of the Islamic State, executing the operation in
response to calls to target citizens of coalition nations‚Äù, but has been unable to prove officials
were in any contact with Masood. Describing him as a terrorist, the Metropolitan Police
confirmed that he acted alone. Later on March 27th, Neil Basu, Deputy Assistant Commissioner
of the Metropolitan Police and Senior National Coordinator for UK Counter-Terrorism Policing,
announced that Masood clearly had an interest in Jihad, but they have found no evidence he
was linked with any terrorist Organization. Six property raids and eleven arrests were made, but
no charges were filed and all suspects were released without charges.
